cns/ipampool/monitor.go

```diff
@@ -146,7 +154,9 @@ func (pm *Monitor) reconcile(ctx context.Context) error {
 	allocatedIPs := pm.httpService.GetPodIPConfigState()
 	state := buildIPPoolState(allocatedIPs, pm.spec)
 	logger.Printf("ipam-pool-monitor state %+v", state)
-	observeIPPoolState(state, pm.metastate)
+
+	labels := []string{subnet, subnetCIDR}
+	observeIPPoolState(state, pm.metastate, labels)
```

Comment:
can you inline this - we don't need a local variable `lables` - and remove the whitespace?

Reply:
Made this change. I have pushed the necessary changes.